aws/eks-anywhere
# Remove unnecessary validations for upgrade management components

*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*
- Manual testing
- Running E2E tests
```
--- PASS: TestVSphereKubernetes128UpgradeManagementComponents (747.19s)
PASS
```
```
--- PASS: TestDockerKubernetesUpgradeManagementComponents (615.07s)
PASS
```
```
--- PASS: TestCloudStackKubernetes128UpgradeManagementComponents (854.62s)
PASS
```

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
